test/sql/timezone/icu_calendar_operations.test

```diff
@@ -0,0 +1,22 @@
+# name: test/sql/timezone/icu_calendar_operations.test
+# description: Test the ICU calendar interface
+# group: [timezone]
+
+require icu
+
+# Normalise the testing locale
+statement ok
+SET Calendar = 'hebrew';
```

Comment:
We should test a year in the Hebrew calendar that has Adar II (the 13th month.) [Mid-March 2024](https://www.hebcal.com/holidays/rosh-chodesh-adar-ii-2024) is an example.

Reply:
(Also, calendar name are case-insensitive, so we should probably capitalise "Hebrew")

Reply:
I used 2020 because it has the 13th month:

```sql
set calendar='hebrew';
select timestamptz '2020-06-01' + interval (230) day as res;
┌───────────────────────────┐
│            res            │
│ timestamp with time zone  │
├───────────────────────────┤
│ 2020-13-24 00:00:00+00:17 │
└───────────────────────────┘
```

Of course the years mean something different as well (it is currently the year 5785 in the hebrew calendar) - 2020 is actually several hundred years B.C.

Reply:
Ah I forgot how that gets parsed - nicely done!